frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -528,9 +528,16 @@ common_settings.document = {
 
 local metadata_folder_str = {
     ["doc"] = _("book folder"),
-    ["dir"] = "koreader/docsettings",
+    ["dir"] = "koreader/docsettings/",
 }
 
+local metadata_folder_help_text = _([[
+Book view settings, reading progress, highlights, bookmarks and notes (collectively known as metadata) are stored in a separate folder named <book-filename>.sdr (".sdr" meaning "sidecar").
+
+You can decide between two locations where these will be saved:
+- alongside the book file itself (the long time default): these sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you do directories synchronisation or backups, your settings will be part of them (which might be what you want, or not).
```

Comment:
We use "synchronization" elsewhere in the program.
```suggestion
- alongside the book file itself (the long time default): these sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you perform directory synchronization or backups, your settings will be part of them.
```